framework/contracts/native/module-factory/src/commands.rs

```diff
@@ -21,6 +23,30 @@ use crate::{
     state::*,
 };
 
+#[derive(cosmwasm_schema::serde::Serialize, cosmwasm_schema::serde::Deserialize)]
+#[serde(crate = "::cosmwasm_schema::serde")]
+struct MockStandaloneMsg {
+    // Fill base for contract
+    #[serde(default, deserialize_with = "deserialize_some")]
+    base: Option<Option<standalone::BaseInstantiateMsg>>,
+
+    // Forward other fields
+    // https://serde.rs/attr-flatten.html#capture-additional-fields
+    #[serde(flatten)]
+    extra: HashMap<String, Value>,
+}
+
+// We need this to distinguish between `"base": null` and missing field
+// https://github.com/serde-rs/serde/issues/984#issuecomment-314143738
+// Any value that is present is considered Some value, including null.
+fn deserialize_some<'de, T, D>(deserializer: D) -> Result<Option<T>, D::Error>
+where
+    T: cosmwasm_schema::serde::Deserialize<'de>,
+    D: cosmwasm_schema::serde::Deserializer<'de>,
+{
+    cosmwasm_schema::serde::Deserialize::deserialize(deserializer).map(Some)
+}
+
```

Comment:
At first thought it's fine if user fills base themself, but after adding `AccountBase` inside it became annoying because if account is not created yet you need to get address of proxy or manager with instantiate2 first.
Alternative to this solution we could inherit `{base, module}` structure from apps (we will still need to verify base is `null` if we want to support arbitrary messages as well)

Reply:
Why would they need to include the AccountBase? That can be interpolated from the admin right?

Reply:
I.e. query query account_base from the caller (which should be a manager)

Reply:
> I.e. query query account_base from the caller (which should be a manager)

Caller of instantiate msg is module-factory.

Do you know if "contract admin" set during instantiation?